MoltenVK/MoltenVK/GPUObjects/MVKRenderPass.mm

```diff
@@ -767,7 +767,17 @@ static uint32_t getNextViewMaskGroup(uint32_t viewMask, uint32_t* startView, uin
 #pragma mark -
 #pragma mark MVKRenderPass
 
-VkExtent2D MVKRenderPass::getRenderAreaGranularity() { return { 1, 1 }; }
+VkExtent2D MVKRenderPass::getRenderAreaGranularity() {
+#if MVK_IOS_OR_TVOS || MVK_MACOS_APPLE_SILICON
+    if (_device->_pMetalFeatures->tileBasedDeferredRendering) {
+        // This is the tile area.
+        // FIXME: We really ought to use MTLRenderCommandEncoder.tile{Width,Height}, but that requires
+        // creating a command buffer.
+        return { 32, 32 };
+    }
+#endif
+    return { 1, 1 };
+}
```

Comment:
Just trying to think of ways to make this not-hardcoded.

Is granularity settable in Vulkan the way it is via `MTLRenderPassDescriptor.tile{Width,Height}`? Or do we just need a way of describing a value set by the system? I couldn't find it as settable within Vulkan...but I may have missed something?

And do we know if the default value of `MTLRenderCommandEncoder.tile{Width,Height}` and/or `MTLRenderPassDescriptor.tile{Width,Height}` are fixed for the GPU, or could they change renderpass by renderpass, based on FB bounds? Wondering if we could create a temp `MTLRenderPassDescriptor` or `MTLRenderCommandEncoder` at `MVKDevice` creation time and read the values?

Reply:
> Is granularity settable in Vulkan the way it is via `MTLRenderPassDescriptor.tile{Width,Height}`? Or do we just need a way of describing a value set by the system? I couldn't find it as settable within Vulkan...but I may have missed something?

I couldn't find one either. So we just need a way of describing the value that was set by Metal.

> And do we know if the default value of `MTLRenderCommandEncoder.tile{Width,Height}` and/or `MTLRenderPassDescriptor.tile{Width,Height}` are fixed for the GPU, or could they change renderpass by renderpass, based on FB bounds?

We don't know. I was afraid it were the latter, so I picked the most conservative value. Maybe we should ask.